### PR TITLE
[fr] #3262 & #3254

### PR DESCRIPTION
**Pull request by cake17**

### CODE REVIEW

File: fr/core-libraries/caching.rst [hunk 1]
```diff
@@ -38,10 +38,10 @@ votre propre système de mise en cache. Les moteurs de cache intégrés sont:
   est une extension PHP qui fournit des fonctionnalités similaires à APC.
 * ``MemcachedEngine`` Utilise l'extension
   `Memcached <http://php.net/memcached>`_. Il est aussi une interface avec
-  memcache mais il fournit une meilleur performance.
+  Memcached mais il fournit une meilleur performance.
```
Comment (antograssiot) [inline fr/core-libraries/caching.rst]: This sentence doesn't make sense anymore and should be removed I think. It's coming from 2.x doc where MemacheEngine was an option.
http://book.cakephp.org/2.0/en/core-libraries/caching.html

File: fr/views.rst [hunk 2]
```diff
@@ -567,44 +567,10 @@ fournissez une valeur unique de la clé cache en utilisant le format suivant::
...
-.. code-block:: php
-
-    <h2>Latest Posts</h2>
-    <?php $posts = $this->requestAction('posts/index?sort=created&direction=asc&limit=5'); ?>
-    <ol>
-    <?php foreach ($posts as $post): ?>
-          <li><?= $post['Post']['title'] ?></li>
-    <?php endforeach; ?>
-    </ol>
+Si vous avez besoin de plus de logique dans votre element, comme des données
```
Comment (HavokInspiration) [inline fr/views.rst]: Do we keep "element" or do we go with "élément" ?
These are the "edge" cases where I want to put the word in French but that still refers to a real concept in Cake...

Comment (cake17) [inline fr/views.rst]: I prefer `element` to keep english terms, but I can change if necessary

File: fr/core-libraries/caching.rst (hunk 1 above)
Comment (cake17) [inline fr/core-libraries/caching.rst]: you're right, I'ill remove it for all versions

Comment (cake17) [inline fr/core-libraries/caching.rst]: By the way, there is also a sentence in [the api](http://api.cakephp.org/3.0/class-Cake.Cache.Engine.MemcachedEngine.html) that might need a change::
    
    Main advantage of this Memcached engine over the memcached engine... ?

File: fr/views.rst (hunk 2 above)
Comment (HavokInspiration) [inline fr/views.rst]: No no, no problem with me, I was just wondering where our limit was :)

Comment (cake17) [inline fr/views.rst]: :) tu as raison, à chaque fois je me pose la question ! Dans ce cas, je le sens plus sans accent parce que, comme tu le disais, c'est un élément technique de CakePHP. Mais bon, d'un autre côté on a gardé des `vue` (pour `view`) un peu partout dans la docs.... :smile:

**Comment by antograssiot** [comment]
:+1:

File: fr/views.rst (hunk 2 above)
Comment (HavokInspiration) [inline fr/views.rst]: Ca vaudrait peut être le coup d'uniformiser.
Mais c'est un travail colossal. Si au moins on peut ne pas en ajouter, ça serait déjà bien.

Comment (cake17) [inline fr/views.rst]: Au passage, une ébauche de [lexique](https://github.com/cakephp-fr/docs-cakephp/wiki/Lexique-de-traduction-des-termes-techniques)

**Pull request merged**